mmdb/init.lua

```diff
@@ -320,10 +333,10 @@ getters[32] = {
 function geodb_methods:search(bits, node)
 	node = node or 0
 	local seen = { [node] = true }
-	for _, direction in ipairs(bits) do
+	for i = 1, #bits do
```

Comment:
Okay, but there is a slight change in behaviour: `ipairs` always stops at first `nil`; while your code is undefined in the presence of `nil`s.

I'm happy to make the change, but please include it in a different commit (should possibly even be in a different PR, but I suppose we can sneak it in)